src/main/java/uk/gov/hmcts/reform/preapi/repositories/ShareRecordingRepository.java

```diff
@@ -9,4 +9,5 @@
 
 @Repository
 public interface ShareRecordingRepository extends JpaRepository<ShareRecording, UUID> {
+    boolean existsByIdAndDeletedAtIsNull(UUID id);
```

Comment:
I don't think this function is being used. Is this going to be used or can it be deleted?

Reply:
Removed